torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -196,6 +196,11 @@ def convert(self):
         raise NotImplementedError('`convert` is not implemented. Please, use '
                                   '`torch.ao.utils.convert` instead.')
 
+    def get_module_pruned_outputs(self, module):
+        r"""Returns the set of pruned indices of module"""
+        assert parametrize.is_parametrized(module)  # can only get pruned indices of pruned module
+        return module.parametrizations.weight[0].pruned_outputs
```

Comment:
How do we know that this attr 'weight' is present in module.parametrizations?

Reply:
I'll add a check that the module is in the pruner's `module_groups`. So far (for the sparsifier also), we assume that this is the only parametrization attached to `weight`.